src/components/card/Card.tsx

```diff
@@ -75,20 +29,38 @@ export const Card = ({
   label,
   spacing = theme.space24,
 }: CardProps) => (
-  <StyledCard $spacing={spacing} className={className || ''}>
+  <div
+    className={`${styles.card} ${className || ''}`}
+    style={{ padding: spacing }}
+  >
     {label && (
-      <CardHeader $spacing={spacing}>
+      <div
+        className={styles.cardHeader}
+        style={{
+          margin: spacing ? `calc(${spacing} * -1)` : theme.spaceNegative24,
+          marginBottom: spacing || theme.space24,
+          padding: spacing || theme.space24,
```

Comment:
We discussed about this, and we might need to refactor this to use css variable instead.